csv-samples/readme.md

```diff
@@ -0,0 +1,64 @@
+# CSV Parser Node Samples
+## General instructions
+This directory contains examples of the use of the CSV Parser node in App Connect integrations.
+Each example comes with a brief description of the scenario covered, and a flow file that implements the scenario.
+
+You'll find the following resources useful for all of them
+* [How to use the CSV Parser Node](https://developer.ibm.com/integration/blog/2017/10/18/how-to-use-the-csv-parser-node-in-app-connect/)
+* [How to upload integration flow files](https://developer.ibm.com/integration/blog/2017/09/29/exporting-importing-flows-ibm-app-connect/)
+
+## Use-cases
+### CSV file from Box used to add/update contacts to Salesforce
+#### Overview
+In this scenario, an App Connect flow exposes an API that
+allows you to identify a file in box via its file URL.
+The file contains a table of 'contact' details, which is used to
+update contact information in salesforce.
+
+#### To-do list
+1. Upload your CSV file to Box
+  * You can use `samplecontacts.csv` to get started
+2. Upload the flow file `CSV-File-in-Box-to-Salesforce-contacts.yaml` (see *Resources* below)
+  * This will create a new integration, which you will see in the App Connect dashboard
+3. Open the newly created application
+  * Review the configuration of each node, so that you understand how it has been configured
+  * If you do not already have a connection to Box - Click on the Box node, and connect to Box
+  * If you do not already have a connection to Salesforce - Click on the Salesforce node, and connect to Salesforce
+4. Save and start the integration
+5. Click on the 'manage' tab for your API, and set up the API authentication
+  * Near the bottom of the page, you'll see a link to a dedicated 'API Portal' for consumers of your API to use
```

Comment:
The API Portal et al is only available in App Connect on Bluemix.  Its not in appconnect.ibmcloud.com
On Bluemix, the instructions given might be adapted for the sample:
To try the sample, 
- Scroll down to 'Sharing Outside of Bluemix Organization'
- Click 'Create API Key' and then copy the 'API KEY'
- Click the 'API PORTAL LINK' link. 
- To explore your API in the portal and invoke it, click 'Try it'
- For the Parameter 'data', click 'Generate' to create the API request body template

Reply:
Done